apps/class-solid/src/components/ExperimentConfigForm.tsx

```diff
@@ -50,8 +50,8 @@ export function ExperimentConfigForm({
   onSubmit,
 }: {
   id: string;
-  config: ClassConfig;
-  onSubmit: (c: ClassConfig) => void;
+  config: Partial<ClassConfig>;
```

Comment:
Tracking only the fields that are explicitly set makes it easier to see which fields deviate from defaults, both in form and in the config file.